### ISSUE TITLE
camelCase attrs not working always

### ISSUE THREAD

**Issue by alexandernst** [clientIO/joint#1421]
I'm playing around with `joint.elementTools.Boundary` and I tried to set the stroke dasharray attribute using the camelCase syntax, but it didn't work.

```
joint.elementTools.FTABoundary = joint.elementTools.Boundary.extend({
  attributes: {
    ...
    strokeDasharray: 8,  <-- doesn't work
    "stroke-dasharray": 8  <-- it works
    ...
  },
```

**Comment by alexandernst** [comment]
Another place where this occurs:

```
joint.shapes.standard.Link.define("app.Link", {
  attrs: {
    line: {
      stroke: "red",
      strokeWidth: 2, // <---- this DOES work
      targetMarker: {
        type: "path",
        d: "M -5 -5 C 2 -5 2 5 -5 5",
        fill: "none",
        strokeWidth: 2, // <---- this DOESN'T work
      }
    },
  },
});
```

**Comment by alexandernst** [comment]
un-stale

**Comment by kumilingus** [comment]
Fixed in `v4.0.0`.
